lib/components/narrative/itinerary-carousel.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component, PropTypes } from 'react'
+import { Button } from 'react-bootstrap'
+import { connect } from 'react-redux'
+import SwipeableViews from 'react-swipeable-views'
+
+import { setActiveItinerary, setActiveLeg, setActiveStep } from '../../actions/narrative'
+import Icon from './icon'
+import NarrativeItinerary from './narrative-itinerary'
+import Loading from './loading'
+import { getActiveSearch } from '../../util/state'
+
+class ItineraryCarousel extends Component {
+  state = {}
+  static propTypes = {
+    itineraries: PropTypes.array,
+    pending: PropTypes.bool,
+    activeItinerary: PropTypes.number,
+    hideHeader: PropTypes.bool,
+    setActiveItinerary: PropTypes.func,
+    setActiveLeg: PropTypes.func,
+    setActiveStep: PropTypes.func
+  }
+  _onItineraryClick = () => {
```

Comment:
Can you put a blank line between all these methods?  It's a little hard to read.